### PR TITLE
coreutils: Build a bottle for Linuxbrew

### PR DESCRIPTION
**Pull request by iMichka**
This is an automated pull request to build a new bottle for linuxbrew-core
based on the existing bottle block from homebrew-core.

### CODE REVIEW

**Comment by iMichka** [comment]
> configure: error: you should not run configure as root (set FORCE_UNSAFE_CONFIGURE=1 in environment to bypass this check)

File: Formula/coreutils.rb [hunk 1]
```diff
@@ -34,6 +34,9 @@ class Coreutils < Formula
   conflicts_with "truncate", :because => "both install `truncate` binaries"
 
   def install
+    # Fix configure: error: you should not run configure as root
+    ENV["FORCE_UNSAFE_CONFIGURE"] = "1" unless ENV["CI"]
```
Comment (dawidd6) [inline Formula/coreutils.rb]:
```ruby
ENV["FORCE_UNSAFE_CONFIGURE"] = "1" unless ENV["CI"].nil?
```
Maybe?

Comment (iMichka) [reply Formula/coreutils.rb]: @sjackman what is the preferred ENV variable I should use to detect that we are on the azure CI?

Comment (iMichka) [reply Formula/coreutils.rb]: The variable is fine. We need to change the unless with an if ...

Comment (issyl0) [reply Formula/coreutils.rb]: It looks like brew test-bot's [code sets a `HOMEBREW_AZURE_PIPELINES` envvar](https://github.com/Homebrew/homebrew-test-bot/blob/master/cmd/brew-test-bot.rb#L1659)?

Comment (sjackman) [reply Formula/coreutils.rb]: `ENV["CI"]` is good.

**Pull request closed without merging**